config/barong.yml

```diff
@@ -16,3 +16,6 @@ document_types:
   - Utility Bill
   - Residental
   - Institutional
+data_storage_titles:
```

Comment:
I would set the default value as empty array here to prevent accidentally activating user storage.

Reply:
okay. but i will keep possible options as a comment to show possible functionality